### PR TITLE
Update/audit DNS hosting transfer article

### PR DESCRIPTION
**Pull request by catlook**
Closes issue #2144 
Kelly please edit for me!

### CODE REVIEW

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-d569fd2540/how-to/).

**Comment by kmsholcomb** [comment]
Hi @catlook. Edits are attached (Word doc and PDF copy of it). Please let me know if you have any questions. Thanks!

[Transfer your DNS hosting to Rackspace_editsKH.pdf](https://github.com/rackerlabs/rackspace-how-to/files/1174008/Transfer.your.DNS.hosting.to.Rackspace_editsKH.pdf)
[Transfer your DNS hosting to Rackspace_editsKH.docx](https://github.com/rackerlabs/rackspace-how-to/files/1174009/Transfer.your.DNS.hosting.to.Rackspace_editsKH.docx)

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-33acb46e25/how-to/).

File: content/cloud-servers/transferring-your-dns-hosting-to-rackspace.md [hunk 1]
```diff
@@ -1,28 +1,65 @@
 ---
 permalink: transferring-your-dns-hosting-to-rackspace/
-audit_date:
+audit_date: 2017-07-26
```
Comment (stephamon) [inline content/cloud-servers/transferring-your-dns-hosting-to-rackspace.md]: The date needs single quotes around it, like in `created_date` and `last_modified_date`

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-25a3854bf9/how-to/).

**Comment by stephamon** [comment]
👍

**Pull request merged**